report/clients/psi.js

```diff
@@ -36,7 +36,12 @@ import {Util} from '../renderer/util.js';
  *
  * @param {LH.Result | string} LHResult The stringified version of {LH.Result}
  * @param {Document} document The host page's window.document
- * @return {{scoreGaugeEl: Element, perfCategoryEl: Element, finalScreenshotDataUri: string|null, scoreScaleEl: Element, installFeatures: Function}}
+ * @return {{
```

Comment:
was this necessary? last I checked this looks like trash in vscode syntax highlighting (EDIT: just check it doesn't look bad now so meh. it is a departure from how we do jsdoc everywhere else though there is an argument it is more readable)